azurerm/internal/services/datafactory/data_factory_trigger_blob_event_resource.go

```diff
@@ -92,6 +92,11 @@ func resourceDataFactoryTriggerBlobEvent() *pluginsdk.Resource {
 				},
 			},
 
+			"activated": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
```

Comment:
should this be defaulted to true?

Reply:
Yes, to have best (functional) backward compatibility we probably should indeed. Consider it done!